metadata/ComputerVision__8__metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "changeLog": "Release v2023.10.1",
+  "changeLog": "Release v2023.10.2",
```

Comment:
This is the correct version, we are trying to keep it in sync with the CU version number. Not 100% clear to me where the next CU version number is documented, maybe Ioana Ungureanu knows, I get it from what appears to be the latest release branch name of the AS repos, currently this one https://github.com/UiPath/service-fabric-packaging/tree/release/v2023.10.2

If we don't skip any CU then incrementing the previous version should be OK :D

Reply:
If you find out where the future release numbers are documented please consider adding that to our Confluence docs.